### DISCUSSION TITLE
Hex bolt shaft option

### DISCUSSION THREAD

**Discussion by doudar** [doudar/SmartSpin2k#267] [Show and tell]
Mostly for @eMadman and not tested outside of cad yet (I'm sure he will soon though), files have been included for a metal shaft using a 1.5" 5/16" bolt in place of the printed shaft we've been using. If it all fits and works, this should be a much more robust solution as well as preventing any knob cup wobble from bent shafts in the past. 

Files are [Hex 40T Gear](https://github.com/doudar/SmartSpin2k/tree/develop/Hardware/Case%20Version2/Gears/Spur%20gear%201M%2040T%20Hex.STL) and [Hex Knob Cup](https://github.com/doudar/SmartSpin2k/tree/develop/Hardware/KnobCups/Knob_Cup_Large_HEX.STL)

<img src="http://github.com/doudar/SmartSpin2k/tree/develop/Pictures/HexBolt.JPG" alt="HexBolt"/> 

![image](https://user-images.githubusercontent.com/17362216/144662581-88561579-c0e4-4876-b424-a65764e419c1.png)

**Comment by eMadman** [comment]
looks good!  I'll have to make a trip out to home depot for the bolts.  Everything I've got is metric and maxes out at M5.  Likely would be stronger than the plastic itself.  Looks like a captive nut on the inside of the cup might not be a bad idea.  Might save having to deal with a socket wrench there.  Embarrassed as I am to admit it, I'm a hobbyist maker without a useful socket set.  Ha!

**Reply by doudar** [reply]
Grab an M8 while you’re at it. It should work as well. I think they’re interchangeable except for the threads. Just don’t go longer than 1.5” or your knob insert won’t fit.

**Reply by eMadman** [reply]
Grainger seems to have the parts and I happen to need more machine scres.  I'll toss these into my order.  

For the IC4 mod, I'm trying to combine the cup and insert.  I think I see a happy path to do that with this design and have a captive jam nut with embedded in it.  This will certainly end up being the strongest part in the enclosure.

**Comment by eMadman** [comment]
Making some changes to the CAD - the 5/16"x1.5" nuts and bolts I have from grainger have bigger heads than in the model.  I think I have the changes to where I like it... I'll need to go back into the cup file and provide an updated one.  I ended up going to town and merging this with the IC4 insert to make for faster printing.  I should have one printed in the next day or two

**Reply by doudar** [reply]
That’s interesting. What’s the size difference? Are the nuts a different dimension than the bolt head?

**Reply by eMadman** [reply]
bolt and nut are both 12.5mm across the flats.  I have 5mm deep for the bolt head, 6.75mm deep for the nut.  

In the cad, it's 12.1mm across the flats.  I also neglected to get jam nuts so I added space for a standard nut on the knob cup

Tolerances were a bit too exact.  My printer can usually do down to 0.2mm with no issues, closer than that, there's a chance it'll be too tight.  

Going to print it now... should be ~2-2.5 hours

**Reply by eMadman** [reply]
Initial results:  wasted a lot of time using my 0.6mm nozzle.  Couldn't get a good mesh on the gears.  I'm well tuned for larger objects on 0.6mm, but the gears gave me issues.

Attempt 2 with 0.4mm nozzle:  I love the heft of it and it's rock solid.  I'm using blue loctite to prevent accidental loosening.

**Reply by doudar** [reply]
I'm currently doing my first print on the hex shaft. It should be done in the next hour. 

Interesting you are having issues with the gear mesh with the .6 nozzle. I don't seem to have any issues with it. What's your layer height and which slicer are you using?

**Reply by eMadman** [reply]
0.4mm layers (also tried 0.3mm) and Superslicer (latest version).  I think the bigger issue is the blobs/zits/stringing I'm getting between the teeth.  I tried using a hobby knife to clean it up, but it still found reason to skip often.  

I started using the big nozzle in the last few months so I know there's a lot more tuning I can do

**Reply by doudar** [reply]
I’m doing .3mm @ 70mm/s @ 250c for petg with the .6mm nozzle. I’d like to go to .4mm height but I run into volumetric issues- the E3D heat block can’t keep up.
